pkg/collector/providers/file.go

```diff
@@ -123,13 +102,54 @@ func (c *FileConfigProvider) String() string {
 	return "File Configuration Provider"
 }
 
-func collectDir(parentPath string, folder os.FileInfo) []check.Config {
+func (c *FileConfigProvider) collectEntry(entry os.FileInfo, path string, checkName string) (check.Config, bool, error) {
+	conf := check.Config{}
+	entryName := entry.Name()
+	ext := filepath.Ext(entryName)
+	isDefault := false
+
+	// skip config files that are not of type:
+	//  * check.yaml, check.yml
+	//  * check.yaml.default, check.yml.default
+	if ext == ".default" {
```

Comment:
you could use a select instead of ifs, as [cases can be expressions and not only constants](https://gobyexample.com/select)

Reply:
Not sure it's the right use case here, each `if` block can have impact on the one bellow

Reply:
Yeah, my bad